src/LLVM_intrinsics.jl

```diff
@@ -0,0 +1,530 @@
+# LLVM operations and intrinsics
+module LLVM
```

Comment:
I would prefer this being called `Intrinsics` or some such.

In the long run I would love some of the more complicated pieces of string manipulation to be converted to LLVM.jl
so the naming of this module would be conflicting, and for me it is most similar to `Core.Intrinsics`.
(There have been some discussions around removing the string based llvmcall from Base entirely, since it is not a stable API)